governance/roles.md

```diff
@@ -11,9 +11,9 @@ The following is the current listing of member roles:
 * [Chairs](#role-of-chairs)
 * [Technical Leads](#role-of-technical-leads)
 * [Project Leads](#role-of-project-leads)
-* [TAG Representatives](#role-of-tag-representatives)
-* [Team leads](#role-of-team-leads)
-* [Group Members](#role-of-members)
+* [TAG Representatives](#role-of-environmental-sustainability-tag-representatives)
+* Team leads
+* Group Members
```

Comment:
There are no sections corresponding to "Team leads" and "Group members" AFAICT (and the anchors don't exist), so I removed the anchors.